@Test public void should_let_throwing_extractor_runtime_exception_bubble_up(){
  assertThatExceptionOfType(RuntimeException.class).isThrownBy(() -> assertThat(employees).extracting(employee -> {
    if (employee.getAge() > 100)     throw new RuntimeException("age > 100");
    return employee.getName().getFirst();
  }
)).withMessage("age > 100");
}
